components/ui/SpringAnimation.tsx

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import { animated, useSpring } from "react-spring";
+
+export const SpringAnimation: React.FC<{
```

Comment:
do you have a better name for this? how could this animation be used? spring animation is pretty generic. imagine we had 5 different animations all using the spring library, how would we know what this was for? 

Maybe use a name that hints at how this animation is being used.